kolibri/plugins/management/templates/management/management.html

```diff
@@ -1,4 +1,19 @@
-{% extends "kolibri/base.html" %}
-{% block content %}
-<span>The management app!</span>
-{% endblock %}
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Kolibri</title>
+  </head>
+  <body>
+    <main v-ref:main></main>
```

Comment:
It allows the component to be referenced by the given name `main` from a Vue instance containing it: `vm.$refs.main`. In production it seems unlikely that we will use this. But for testing it allows components to be referenced "nicely" -- for instance if I change the relative order of subcomponents and I retain the same `v-ref` name then tests will be unaffected.